### DISCUSSION TITLE
.env for WSL

### DISCUSSION THREAD

**Discussion by vicshi06** [smartcontractkit/full-blockchain-solidity-course-py#1555] [Q&A] [answered]
Hi,

I am using WSL, and I am stuck at brownie not being able to detect the INFURA_PROJECT_ID environmental variable I assigned in the `.env` file. I also tried to set up the same environmental variable in the windows with the same name and value, but it was no use.  How do I solve this? 

The code can run successfully in the Development network.

**Comment by cromewar** [comment, marked as answer]
Hello @vicshi06 this happens when the current terminal is not able to recognize the .env file, even if you set it up on `brownie-config` so the easiest way to get this trough is just copy and paste the .env variables to the terminal with `export` keyword.

So on your WSL terminal copy and paste:

`export INFURA_PROJECT_ID= yourId`

This should solve your problem, cheers.
